parameter/ParameterBlackboard.cpp

```diff
@@ -94,6 +94,25 @@ void CParameterBlackboard::readBuffer(void* pvDstData, size_t size, size_t offse
     readInteger(pvDstData, size, offset);
 }
 
+// Element access
+void CParameterBlackboard::writeBytes(const std::vector<uint8_t>& bytes, uint32_t offset)
+{
+    assertValidAccess(offset, bytes.size());
+
+    auto dest_first = atOffset(offset);
```

Comment:
I do not realy get why this intermediate variable is usefull. But if you like it, keep it. It is a mater of style.